dvc/commands/experiments/__init__.py

```diff
@@ -93,8 +93,9 @@ def add_rev_selection_flags(
         dest="num",
         metavar="<num>",
         help=(
-            f"{command} experiments from the `--rev` baseline and from `num` "
-            "commits before it (first parents). Give a negative value to "
-            "include all first-parent commits."
+            f"{command} experiments from the last `num` commits "
+            "(first parents) starting from the `--rev` baseline. "
+            "Give a negative value to include all first-parent commits "
+            "(similar to `git log -n`)."
```

Comment:
I don't think we actually need to mention `git log -n` here

Reply:
It is OK for the help text to be different between the cli and the website?

Reply:
Since it's already been added to the other docs it's probably fine to leave it in. I'm not sure that it really needs to be mentioned in the docs either, but it's not that big of a deal either way.

Reply:
@karajan1001 For future reference, it is intended that there can be differences between the CLI and the website. Often, it makes sense to have more detail in the website than in the help text. That's what I was suggesting in https://github.com/iterative/dvc/issues/7444#issuecomment-1062923980, and I think the suggestion from @pmrowla makes sense as well (especially because `git log -n` with negative numbers is undocumented and there are behavioral differences between them since dvc only shows first-parent ancestors).

Not a big deal that requires another PR, but just so you know they don't have to match. cc @jorgeorpinel